test/unit/generator/content/schedule/part3/index.test.js

```diff
@@ -7,8 +7,14 @@ const getReductionSummary = require('../../../../../../app/generator/content/sch
 jest.mock('../../../../../../app/generator/content/schedule/part3/get-top-up-summary')
```

Comment:
We have changed this functionality and need to change the tests for it.

This function no longer directly calls `getTopUpSummary`, `getRecoverySummary` but rather just `getSummary`.

The functionality in this should be moved to getSummary unit test